prusti-specs/src/specifications/preparser.rs

```diff
@@ -259,13 +32,10 @@ impl Parse for Arg {
     }
 }
 
-/// The representation of all arguments to `forall`
-/// (for example `a: i32, b: i32, c: i32`)
```

Comment:
Please keep the documentation for the various `structs`.

Reply:
Addressed by c77b6d0b1638c36b81eddb1e87af37c3f1fff212